### PR TITLE
docs: update 'Testing redux' in CONTRIBUTING.md

### PR DESCRIPTION
**Pull request by a1012112796**
try simplfy it to make it more easy for contributor to follow it to test their work by themselves.

Thanks to @eider @6543 @zeripath's help

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -164,8 +164,8 @@ help:
 	@echo " - revive            run revive linter"
 	@echo " - misspell          check for misspellings"
 	@echo " - vet               examines Go source code and reports suspicious constructs"
-	@echo " - test              run unit test"
-	@echo " - test-sqlite       run integration test for sqlite"
+	@echo " - test[\#TestSpecificName]    			run unit test"
+	@echo " - test-sqlite[\#TestSpecificName]       run integration test for sqlite"
 	@echo " - pr#<index>        build and start gitea from a PR with integration test data loaded"
```
Comment (6543) [inline Makefile]: I think you have to add space for the oter entrys too to have the same aligment

Comment (a1012112796) [reply Makefile]: I wander whether It's a good style ...🤔

image

**Review by 6543 — APPROVED** [review]

File: Makefile [hunk 2]
```diff
@@ -144,29 +144,29 @@ include docker/Makefile
...
+	@echo " - fomantic                         build fomantic files"
+	@echo " - generate                         run \"go generate\""
+	@echo " - fmt                              format the Go code"
+	@echo " - generate-swagger                 generate the swagger spec from code comments"
+	@echo " - swagger-validate                 check if the swagger spec is valid"
+	@echo " - golangci-lint                    run golangci-lint linter"
+	@echo " - revive                           run revive linter"
+	@echo " - misspell                         check for misspellings"
+	@echo " - vet                              examines Go source code and reports suspicious constructs"
+	@echo " - test[\#TestSpecificName]    	   run unit test"
```
Comment (zeripath) [inline Makefile:167]:
```suggestion
	@echo " - test[\\#TestSpecificName]    	   run unit test"
```

File: Makefile (hunk 1 above)
Comment (6543) [reply Makefile]: depending on the window size ...

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -71,23 +71,21 @@ to make sure your changes don't cause regression elsewhere.
...
-- Change into the base directory of your copy of the gitea repository,
-  and run `drone exec --event pull_request`.
-- At the moment `drone exec` doesn't support the Docker Toolbox on Windows 10
-  (see [drone-cli#135](https://github.com/drone/drone-cli/issues/135))
-
-The drone version, command line, and disk requirements do change over
-time (see [#4053](https://github.com/go-gitea/gitea/issues/4053) and
-[#6243](https://github.com/go-gitea/gitea/issues/6243)); if you
-discover any issues, please feel free to send us a pull request to
-update these instructions.
```
Comment (bagasme) [inline CONTRIBUTING.md:90]: why did you remove drone requirements?

Comment (CirnoT) [reply CONTRIBUTING.md:90]: It was discussed that we should not have users install full Drone; performing unit and integration tests from Makefile is enough.

**Review by techknowlogick — APPROVED** [review]

**Comment by techknowlogick** [comment]
ping LG-TM
